### DISCUSSION TITLE
Whisper's temp files

### DISCUSSION THREAD

**Discussion by deadnoises** [openai/whisper#1909] [Q&A]
Hello, this is rather mundane but I just want to know where does whisper send the files it creates when transcribing. I'm fighting for disk storage

**Comment by ryanheise** [comment]
By default, whisper outputs files into the current directory (i.e. "`.`"). For more information, run `whisper --help`:

```
  --output_dir OUTPUT_DIR, -o OUTPUT_DIR
                        directory to save the outputs (default: .)
  --output_format {txt,vtt,srt,tsv,json,all}, -f {txt,vtt,srt,tsv,json,all}
                        format of the output file; if not specified, all
                        available formats will be produced (default: all)
```

So if your input file is `audio.mp3`, then by default it will output `audio.txt`, `audio.vtt`, `audio.srt`, `audio.tsv` and `audio.json` into `.` (the current directory). If you only want to output the SRT file, you can use the option `--output_format srt`.

In terms of storage, much larger than these is the model file which is downloaded the first time you run whisper and is stored in different places depending on your operating system, but it might be in `~/.cache/whisper` or `$XDG_CACHE_HOME/whisper`. The default model is `small` which is 462M. You can choose a different location for the model by specifying `--model_dir path/to/your/dir` which can also be found in the `--help` page.
